README.md

```diff
@@ -8,38 +8,38 @@ Make sure you have all the **requirements** above, if not, install/get them.
 
 *The following commands should be executed in a CMD, Bash or Powershell window. To do this, go to a folder on your computer, click in the folder path at the top and type CMD, then press enter.*
 
-Clone the repository and install the requirements (we recommend using a virtual environment like `conda`):
+Clone the repository and install the requirements (we recommend using a Python virtual environment):
 
 ```bash
-pip install -r requirements.txt
+pip install writemore
 ```
 
-Rename `.env.template` to `.env` and fill in your `OPENAI_API_KEY` from the [OpenAI dashboard](https://platform.openai.com/account/api-keys).
+The library requires the `OPENAI_API_KEY` environment variable to be set with an api key obtained from
```

Comment:
I'd still leave some indication on how to set the `.env` for less experienced users

Reply:
Good point - I'll add to the readme for bash/powershell.